jetty-servlet/src/main/java/org/eclipse/jetty/servlet/ServletContextHandler.java

```diff
@@ -356,13 +356,16 @@ protected void startContext() throws Exception
 
         if (_servletHandler != null)
         {
-            // Call decorators on all holders, and also on any EventListeners before
-            // decorators are called on any other classes (like servlets and filters)
+            //Ensure listener instances are created, added to ContextHandler
             if(_servletHandler.getListeners() != null)
             {
                 for (ListenerHolder holder:_servletHandler.getListeners())
                 {             
-                    _objFactory.decorate(holder.getListener());
+                    holder.start();
```

Comment:
Should holders be explicitly started or can they be added as beans to be managed?
Are they currently listed in a Dump?

Reply:
All holders (Servlet,Filter,Listener) are added as unmanaged beans, and listed as such in the dump output.